public/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+  <body>
+    <div id="app">
+      Humble Bundle
+      <div id="charity">
+        <script src="http://localhost:3987/static/js/bundle.js"></script>
```

Comment:
Does putting this script tag inside the parent div render it within that div automatically?